### PR TITLE
#161348747 Update TravisCI to run tests

### PR DESCRIPTION
**Pull request by RachaelNantale**
#### What does this PR do?
Set up TravisCI for continuous integration
#### Description of Task to be completed?
-  Add Sample test for Travis to run
- Add passing build badge to the readme.md

#### What are the relevant pivotal tracker stories?
[#161348747](https://www.pivotaltracker.com/story/show/161348747)

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY')
+SECRET_KEY = config('SECRET_KEY', default='secr3tk3y')
```
Comment (emugaya) [inline authors/settings.py]: Why are you hardcoding default here?

**Review by emugaya — CHANGES_REQUESTED** [review]
Why do you have 36 commits?

**Review by dnuwa — COMMENTED** [review]
remove the unnecessary spacing in the travis.yml and add a new line below line 21

**Comment by RachaelNantale** [comment]
@emugaya  Initially, I was using git ammend but the wrong way. I Kept using git ammend then git pull which created several merges hence 36 commits but I refactored to one commit

**Pull request merged**